docs/infrastructure/index.asciidoc

```diff
@@ -4,83 +4,32 @@
 
 [partintro]
 --
-Use the interactive Infrastructure UI to monitor your infrastructure and
-identify problems in real time. You can explore metrics and logs for common
-servers, containers, and services. 
+The Infrastructure UI enables you to monitor your infrastructure and identify problems in real time.
```

Comment:
@gchaps Do we have a recommendation for `real time` vs `real-time`? I see 11 instances in the kib docs of the former and 14 instances of the latter.

Reply:
@gchaps? According to https://developers.google.com/style/hyphens, I think this is to be left as is (it's a compound modifier after a noun). But if it was "real-time diagnostics" then we should use a hyphen, as it's a compound modifier before a noun. Perhaps that's the distinction?

Reply:
Ahh, that makes sense. Thanks!